tests/v4l2encode.cpp

```diff
@@ -147,23 +148,23 @@ bool feedOneInputFrame(int fd, int index = -1 /* if index is not -1, simple enqu
 
 bool writeOneOutputFrame(uint8_t* data, uint32_t dataSize)
 {
-    static FILE* fp = NULL;
+    static std::ofstream m_ofs;
 
-    if(!fp) {
-        fp = fopen(outputFileName, "w+");
-        if (!fp) {
+    if(!m_ofs.is_open()) {
+        m_ofs.open((outputFileName, std::ofstream::out | std::ofstream::trunc);
+        if (!m_ofs) {
             fprintf(stderr, "fail to open file: %s\n", outputFileName);
```

Comment:
hi, should this line use std::cerr too?

Reply:
It's a little out of scope for these changes.  I agree, we should replace all fprintf usage... but that should be handled in another patch series.